.github/workflows/ci.yml

```
@@ -9,17 +9,28 @@ on:
       - master
 
 jobs:
-  test-ubuntu:
+  test-debian:
```

```suggestion
  test-debian-like:
```

I think the name fits better.

---

Done.